playbooks/common/openshift-cluster/upgrades/upgrade.yml

```diff
@@ -203,7 +215,7 @@
     command: >
       {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
-    when: inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```

Comment:
> After manually changing one to unschedulable? I guess for the test we'd want to run install, manually change one node to unscheduleable and then run upgrade.

I agree that would definitely be re-set to schedulable, though I don't think that's what's described in the bug.

So, in short, I can't reproduce what they're seeing in the bug but I think this is the right thing to do and I hope we can convince QE of that too.